internal/scorecard/alpha/tests/olm.go

```diff
@@ -47,6 +55,23 @@ func CRDsHaveValidationTest(bundle registry.Bundle) scapiv1alpha2.ScorecardTestR
 	r.State = scapiv1alpha2.PassState
 	r.Errors = make([]string, 0)
 	r.Suggestions = make([]string, 0)
+	crds, err := bundle.CustomResourceDefinitions()
+	if err != nil {
+		r.Errors = append(r.Errors, err.Error())
+		r.State = scapiv1alpha2.ErrorState
```

Comment:
we are using FailState instead of ErrorState for reporting back test failures including errors like this.

Reply:
I think the intent is that FailState is when you actually execute the logic that the test is supposed to cover. And ErrorState would be for another failure that makes it impossible to actually execute the test logic.

In this case, I think the correct choice is ErrorState because we aren't able to even check that the CRDs have validation because we failed to even get the CRDs from the bundle.